app/gui2/src/util/crdt.ts

```diff
@@ -5,31 +5,31 @@ import * as Y from 'yjs'
 import type { Opt } from './opt'
 
 export function useObserveYjs<T>(
-  textRef: Ref<Opt<Y.AbstractType<T>>>,
-  observer: (event: T, transaction: Y.Transaction) => void,
+  typeRef: Ref<Opt<Y.AbstractType<T>>>,
+  observer: (_event: T, _transaction: Y.Transaction) => void,
```

Comment:
are these supposed to be unused parameters? (same applies to `useObserveYjsDeep`)

Reply:
Those are not really unused, as I'm passing the observer function to yjs method, which does use them. I'm surprised that eslint complained here, but I see now that it no longer does, so I'll remove that change. Must have been a transient misconfiguration when working on the config.